.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+- 'stable'
+- '5.12.0'
+- '4.6.0'
+services: mongodb
+env:
+  global:
+    - export MONGO_URL=mongodb://localhost/appointment-reminders-node-test
```

Comment:
I don't think you have to explicitly set this up, I'm absolutely sure that this is not **appointment-reminders**. ;)

Please take a look at this https://docs.travis-ci.com/user/database-setup/#MongoDB